### PR TITLE
[documentation] adding background information about trenchboot

### PR DESCRIPTION
**Pull request by garnetgrimm**
Adding background information that helped me get started

### CODE REVIEW

File: documentation/Background.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# 1) Background Info
+### 1.1) What is Trust?
+Trust is when the computer does something on our behalf we have to trust it with our data. A trusted component is that which always behaves in an expected manner.  To make a component "trusted", you must establish some root of trust. A Root of Trust Measurement is the process of measuring (hashing) components on a system to validate their integrity. When establishing a root of trust, there are two primary schools of thought: Static Root of Trust Measurements and Dynamic Root of Trust Measurements.
```
Comment (pietrushnic) [inline documentation/Background.md:3]: @garnetgrimm it would be way easier to review if the document would be wrapped at 80 characters

**Comment by philipandag** [comment]
@garnetgrimm Are you still working on this pull request? The documentation repository is to be archived and we need to close all PRs.

**Comment by philipandag** [comment]
@garnetgrimm I have made a PR to your fork which hopefully makes it ready to be merged to the documentation. Would you mind reviewing it and merging?
https://github.com/garnetgrimm/documentation/pull/1

**Comment by philipandag** [comment]
The change was moved to another PR
https://github.com/TrenchBoot/documentation/pull/29

**Pull request closed without merging**